.github/workflows/tests.yml.j2

```diff
@@ -9,44 +9,20 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-{% set branch = branch_name %}
-{% set current_release = distro_name + '-' + (distro_release|string) %}
-{# Filter supported releases for current branch and create matrix data #}
-{% set matrix_data = [] %}
-{% for rel in supported_releases %}
-  {# Only include releases that match the current branch #}
-  {% if rel.target_branch == branch %}
-    {# Replace current branch in "release" with empty string. This makes name of the check for
-       the current branch always have the same name, so that it can be added to required
-       checks on GitHub. #}
-    {% set release_name = '' if rel.release == current_release else rel.release %}
-    {# Create matrix entry with all required variables:
-         - release: Either empty string (current) or release name (others)
-         - target_branch: The branch this release targets
-         - ci_tag: The actual release name (used for container tags, etc.) #}
-    {% set _ = matrix_data.append({
-      'release': release_name,
-      'target_branch': rel.target_branch,
-      'ci_tag': rel.release
-    }) %}
-  {% endif %}
-{% endfor %}
-
 jobs:
+  define-matrix:
```

Comment:
Same here - not really clear what category of this this will end up running on.

Reply:
The define-matrix workflow has description for the parameters it exposes btw:

 27       target_branch:                                                                               
 28         description: 'Target branch to filter matrix for (empty for all branches)'
 
 I will add the comment in the workflow saying that we filter the matrix with the releases exposes for PR's base branch.